coderadar-plugins/checkstyle-analyzer-plugin/src/main/java/io/reflectoring/coderadar/analyzer/checkstyle/CheckstyleSourceCodeFileAnalyzerPlugin.java

```diff
@@ -93,7 +95,7 @@ private Configuration getConfigurationFromStream(InputStream in) throws Checksty
     return ConfigurationLoader.loadConfiguration(
         new InputSource(in),
         new CheckstylePropertiesResolver(new Properties()), // TODO: pass real properties
```

Comment:
please handle the todo

Reply:
The TODO is not part of this PR.

Reply:
As far as I knwo, everywhere, evene in GUI we use a lower case c in coderadar

Reply:
In the UI it is written with upper case c.